playground/package.json

```diff
@@ -11,23 +11,14 @@
     "name": "Kyma Core & Supporting Team",
     "url": "http://kyma-project.io"
   },
-  "dependencies": {
-    "fiori-fundamentals": "^1.7.1",
-    "react": "^16.9.0",
-    "react-app-rewired": "^2.1.3",
-    "react-dom": "^16.9.0",
-    "react-scripts": "3.1.1",
-    "react-tabs": "^3.0.0",
-    "styled-components": "^4.3.2"
+  "devDependencies": {
+    "@types/react-tabs": "2.3.1",
+    "react-tabs": "3.0.0",
+    "fiori-fundamentals": "^1.7.1"
   },
   "scripts": {
-    "start": "SKIP_PREFLIGHT_CHECK=true react-app-rewired start",
-    "build": "react-scripts build",
-    "test": "echo \"No tests here!\"; exit 0;",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
+    "start": "SKIP_PREFLIGHT_CHECK=true ../node_modules/.bin/react-app-rewired start",
+    "test": "../node_modules/.bin/react-app-rewired test"
```

Comment:
why do we have test inside playgroud

Reply:
and why do we have react-app-rewired installed in root, and not in playground, if we will only ever have it installed inside playground